GitUI/UserEnvironmentInformation.cs

```diff
@@ -67,6 +69,48 @@ public static string GetGitVersionInfo(string? gitVersion, GitVersion lastSuppor
             return gitVersion;
         }
 
+        public static string GetDotnetVersionInfo()
+        {
+            StringBuilder sb = new();
+            Executable dotnet = new(DOTNET_CMD);
+            ArgumentString[] args = new ArgumentString[]
+           {
+                new ArgumentBuilder()
+                {
+                    "--list-runtimes"
+                },
+
+                new ArgumentBuilder()
+                {
+                    "--list-sdks"
```

Comment:
Why do we need the SDK information? Only runtime info is relevant.

Reply:
...and arguably only Microsoft.WindowsDesktop.App is important.
We need to conscious of the length of the querysting which is passed when "create GitHub" issue functionality is invoked.

Reply:
Ok. I could regex the desktop app lines. Along with not doing sdk. Sdk was to catch any odities between runtimes vs sdks.